helm/druid/values.yaml

```diff
@@ -26,32 +26,37 @@ configMap:
   ##
   enabled: true
 
-## Define the key value pairs in the configmap
-configVars:
+## The configurations below would be set as environment variable across all pod
+globalConfig:
   ## DRUID env vars. ref: https://github.com/apache/druid/blob/master/distribution/docker/druid.sh#L29
-  # DRUID_LOG_LEVEL: "warn"
-  # DRUID_LOG4J: <?xml version="1.0" encoding="UTF-8" ?><Configuration status="WARN"><Appenders><Console name="Console" target="SYSTEM_OUT"><PatternLayout pattern="%d{ISO8601} %p [%t] %c - %m%n"/></Console></Appenders><Loggers><Root level="info"><AppenderRef ref="Console"/></Root><Logger name="org.apache.druid.jetty.RequestLog" additivity="false" level="DEBUG"><AppenderRef ref="Console"/></Logger></Loggers></Configuration>
-  DRUID_USE_CONTAINER_IP: "true"
+  DRUID_USE_CONTAINER_IP: 'true'
 
   ## Druid Common Configurations. ref: https://druid.apache.org/docs/latest/configuration/index.html#common-configurations
-  druid_extensions_loadList: '["druid-histogram", "druid-datasketches", "druid-lookups-cached-global", "postgresql-metadata-storage"]'
-  druid_metadata_storage_type: postgresql
-  druid_metadata_storage_connector_connectURI: jdbc:postgresql://postgres:5432/druid
-  druid_metadata_storage_connector_user: druid
-  druid_metadata_storage_connector_password: druid
+  druid_extensions_loadList: '["druid-datasketches", "druid-lookups-cached-global", "postgresql-metadata-storage"]'
+
+  druid_metadata_storage_type: derby
+
   druid_storage_type: local
+
   druid_indexer_logs_type: file
-  druid_indexer_logs_directory: /opt/data/indexing-logs
 
   ## Druid Emitting Metrics. ref: https://druid.apache.org/docs/latest/configuration/index.html#emitting-metrics
   druid_emitter: noop
-  druid_emitter_logging_logLevel: debug
-  druid_emitter_http_recipientBaseUrl: http://druid_exporter_url:druid_exporter_port/druid
+  druid_emitter_logging_logLevel: info
 
 gCloudStorage:
   enabled: false
   secretName: google-cloud-key
 
+hdfs:
+  # Enable hdfs as deep-storage reference: http://druid.apache.org/docs/latest/development/extensions-core/hdfs.html
+  # Need to do:
+  # 1. enabled: true
+  # 2. add `"druid-hdfs-storage"` to `druid_extensions_loadList`
+  # 3. supply configmap's name which contains hdfs-site.xml and core-site.xml
+  enabled: false
+  configMapName: ''
```

Comment:
can you add an example in `hdfs/values.yaml`? It will also be a bonus if you can add configmap template for HDFS in `templates`. That configmap will be deployed only if `.values.hdfs.enabled` is set to true.

Reply:
The configmap is looks like:

```yaml
apiVersion: v1
kind: ConfigMap
data:
  core-site.xml: |
    <balabala something here>
  hdfs-site.xml: |
    <balabala something here>
```

In fact, the configmap would be generate by every deployment of hdfs, which ought to contain the configurations that hdfs client needs when accessing hdfs. I did not supply a template file for that configmap, since when deploying druid with hdfs as deep storage, the configmap would have been in place.